doc/nrf/app_dev/device_guides/nrf70/features.rst

```diff
@@ -51,8 +51,7 @@ The nRF70 Series devices also support the following functionalities:
 * :ref:`Promiscuous reception <ug_nrf70_developing_promiscuous_packet_reception>`: Allows the reception of IEEE 802.11 packets from a connected BSSID when operating in Station mode.
 * :ref:`Offloaded raw transmission <ug_nrf70_developing_offloaded_raw_tx>`: Allows the offloading of raw IEEE 802.11 frame transmission to the nRF Wi-Fi driver.
 * :ref:`Wi-Fi advanced security modes <ug_nrf70_wifi_advanced_security_modes>`: Allows the use of advanced security modes, certificate-based Wi-Fi security, and the Platform Security Architecture (PSA) security framework.
-
-Peer-to-peer support in the form of Wi-Fi Direct® will be available in the future.
+* :ref:`Wi-Fi Direct® (P2P mode) <ug_wifi_direct>`: Allows the establishment of direct device-to-device connections without requiring a traditional access point.
```

Comment:
This entire section might have to be restructured a bit to list:

Wi-Fi mode
Radio test
Offloaded raw TX 

as modes and then following functionalities:

Station
SoftAP
Scan
Raw TX (missing right now)
Monitor
Promiscuous mode
P2P

in the Wi-Fi mode